src/transformers/models/vit_mae/modeling_vit_mae.py

```diff
@@ -757,6 +839,8 @@ def forward(
         output_attentions=False,
         output_hidden_states=False,
         return_dict=True,
+        interpolate_pos_encoding=False,
+        pixel_values=False,
```

Comment:
Same here re not needing pixel_values 

```suggestion
```

Reply:
the same reason as mentioned in the above reply.